src/PKSim.Core/CoreConstants.cs

```diff
@@ -1176,6 +1176,7 @@ public static bool IsVolumeFraction(string parameterName)
                BLOOD_FLOW,
                GFR_SPEC,
                VOLUME_PLASMA,
+               E_GFR
```

Comment:
@Yuri05 I think we want to be ableto see the distribution of target gfr in our population.